src/app/components/account-details/account-details.component.html

```diff
@@ -724,7 +724,7 @@ <h3 class="menu-header">{{
           : (
               walletAccount
             ? (('general.account' | transloco) + '#' + walletAccount.index)
```

Comment:
missing a space before `#` here.

could use a utility function for getting account label perhaps? or like, a pipe, i don't know

i have one on the receive component (but it's needed across at least 4 components so maybe it could be in a single file)
https://github.com/Nault/Nault/blob/bbb79ca600d67b774332c6cd43b7d534e89e214e/src/app/components/receive/receive.component.ts#L172-L180

(which is also missing a space now)

Reply:
Fixed! Yeah maybe a helper function available in all components would be better.